subsys/bootloader/bl_validation/Kconfig

```diff
@@ -52,6 +52,22 @@ config SB_VALIDATION_METADATA_OFFSET
 
 if SECURE_BOOT_VALIDATION
 
+choice SB_VALIDATE_FW
+	prompt "Firmware validation scheme"
+	default SB_VALIDATE_FW_SIGNATURE if IS_SECURE_BOOTLOADER
+	default SB_VALIDATE_FW_HASH# if NETCORE_BOOTLOADER
+	default SB_VALIDATE_FW_NA
+
+	config SB_VALIDATE_FW_SIGNATURE
+		bool "Signature validation (Recommended)"
+
+	config SB_VALIDATE_FW_HASH
+		bool "Hash validation (not secure)"
```

Comment:
Should we add a `depends on BOARD...` here? To avoid misuse? This list should be easy to maintain (I think it will only be nrf53 network core forever to be honest).

Reply:
Or maybe even better, remove prompts, and don't expose this configuration to the user. IMO this is not something the user should be bothered with.

Reply:
Will add

Reply:
Done. It seems `config`s in `choices` must have a prompt so I had to remove the `choice`.